src/lib/blueprint.ts

```diff
@@ -2,14 +2,109 @@ import type { Openapi } from './openapi.js'
 
 export interface Blueprint {
   name: string
+  routes: Route[]
 }
 
+interface Route {
+  name: string
+  path: string
+  description: string | null
+  namespace: Namespace | null
+  endpoints: Endpoint[]
+  subroutes: Route[]
+}
+
+interface Namespace {
+  name: string
+  path: string
+}
+
+interface Endpoint {
+  name: string
+  path: string
+  methods: Method[]
+  semanticMethod: Method
+  preferredMethod: Method
+  description: string | null
+  isDeprecated: boolean
+  deprecationMessage: string
+  parameters: Parameter[]
+  response: Response
+}
+
+interface Parameter {
+  name: string
+  // TODO generate isRequired, isDeprecated, deprecationMessage from https://github.com/seamapi/nextlove/pull/133/files
+  isRequired: boolean
+  isDeprecated: boolean
+  deprecationMessage: string
+  description: string
+}
+
+interface Response {
+  description: string
```

Comment:
I'm not sure if I already commented about this, but let's scope the rest of `Response` out of this PR. So this looks fine for now Implementing this will require introducing a concept of resources which will be it's own discussion.